release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -431,6 +431,13 @@ With this release, you can install {product-title} on Nutanix Cloud Clusters (NC
 
 For more information, see xref:../installing/installing_nutanix/preparing-to-install-on-nutanix.adoc#installation-nutanix-installer-infrastructure-reqs_preparing-to-install-on-nutanix[Infrastructure requirements].
 
+[id="ocp-4-18-installation-and-update-gcp-c4-c4a_{context}"]
+==== Installing a cluster on {gcp-full} using the C4 and C4A machine series
+
+With this release, you can deploy a cluster on {gcp-short} using the C4 and C4A machine series for compute or control plane machines. The supported disk type of these machines is `hyperdisk-balanced`. If you use an instance type that requires Hyperdisk storage, all of the nodes in your cluster must support Hyperdisk storage, and you must change the default storage class to use Hyperdisk storage.
+
+For more information about configuring machine types, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-gcp[Installation configuration parameters for GCP], link:https://cloud.google.com/compute/docs/general-purpose-machines#c4_series[C4 machine series], and link:https://cloud.google.com/compute/docs/general-purpose-machines#c4a_series[C4A machine series].
```

Comment:
[Doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites): When pointing to a link that exists external to Red Hat’s customer portal documentation, add the link attribute, and use the <document_heading_name> (<document_source>)

```suggestion
For more information about configuring machine types, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-gcp[Installation configuration parameters for GCP], link:https://cloud.google.com/compute/docs/general-purpose-machines#c4_series[C4 machine series] (Compute Engine docs), and link:https://cloud.google.com/compute/docs/general-purpose-machines#c4a_series[C4A machine series] (Compute Engine docs).
```